operating/evaluate-security-risks.adoc

```diff
@@ -0,0 +1,287 @@
+|===
+| Search attribute | Policy criteria
+
+| Add Capabilities
+| Add Capabilities
+
+| Annotation
+| Disallowed Annotation
+
+| CPU Cores Limit
+| Container CPU Limit
+
+| CPU Cores Request
+| Container CPU Request
+
+| CVE
+| CVE
+
+| CVE Published On
+| ✕ Dropped
+
+| CVE Snoozed
+| ✕ Dropped
+
+| CVSS
+| CVSS
+
+| Cluster
+| ⟳ Converted to scope
+
+| Component
+| Image Component (name)
+
+| Component Version
+| Image Component (version)
+
+| Deployment
+| ⟳ Converted to scope
+
+| Deployment Type
+| ✕ Dropped
+
+| Dockerfile Instruction Keyword
+| Dockerfile Line (key)
+
+| Dockerfile Instruction Value
+| Dockerfile Line (value)
+
+| Drop Capabilities
+| ✕ Dropped
+
+| Environment Key
+| Environment Variable (key)
+
+| Environment Value
+| Environment Variable (value)
+
+| Environment Variable Source
+| Environment Variable (source)
+
+| Exposed Node Port
+| ✕ Dropped
+
+| Exposing Service
+| ✕ Dropped
+
+| Exposing Service Port
+| ✕ Dropped
+
+| Exposure Level
+| Port Exposure
+
+| External Hostname
+| ✕ Dropped
+
+| External IP
+| ✕ Dropped
+
+| Image
+| ✕ Dropped
+
+| Image Command
+| ✕ Dropped
+
+| Image Created Time
+| Days since image was created
+
+| Image Entrypoint
+| ✕ Dropped
+
+| Image Label
+| Disallowed Image Label
+
+| Image OS
+| Image OS
+
+| Image Pull Secret
+| ✕ Dropped
+
+| Image Registry
+| Image Registry
+
+| Image Remote
+| Image Remote
+
+| Image Scan Time
+| Days since image was last scanned
+
+| Image Tag
+| Image Tag
+
+| Image Top CVSS
+| ✕ Dropped
+
+| Image User
+| ✕ Dropped
+
+| Image Volumes
+| ✕ Dropped
+
+| Label
+| ⟳ Converted to scope
+
+| Max Exposure Level
+| ✕ Dropped
+
+| Memory Limit (MB)
+| Container Memory Limit
+
+| Memory Request (MB)
+| Container Memory Request
+
+| Namespace
+| ⟳ Converted to scope
+
+| Namespace ID
+| ✕ Dropped
+
+| Pod Label
+| ✕ Dropped
+
+| Port
+| Port
+
+| Port Protocol
+| Protocol
+
+| Priority
+| ✕ Dropped
+
+| Privileged
+| Privileged
+
+| Process Ancestor
+| Process Ancestor
+
+| Process Arguments
+| Process Arguments
+
+| Process Name
+| Process Name
+
+| Process Path
+| ✕ Dropped
+
+| Process Tag
+| ✕ Dropped
+
+| Process UID
+| Process UID
+
+| Read Only Root Filesystem
+| Read-Only Root Filesystem
+
+| Secret
+| ✕ Dropped
+
+| Secret Path
+| ✕ Dropped
+
+| Service Account
+| ✕ Dropped
+
+| Service Account Permission Level
+| Minimum RBAC Permission Level
+
+| Toleration Key
+| ✕ Dropped
+
+| Toleration Value
+| ✕ Dropped
+
+| Volume Destination
+| Volume Destination
+
+| Volume Name
+| Volume Name
+
+| Volume ReadOnly
+| Writable Volume
+
+| Volume Source
+| Volume Source
+
+| Volume Type
+| Volume Type
+|===
+
+[id="view-risk-details"]
+== Viewing risk details
+
+When you select a deployment in the *Risk* view, the *Risk Details* open in a panel on the right.
+The *Risk Details* panel shows detailed information grouped by multiple tabs.
+
+include::modules/risk-indicators-tab.adoc[leveloffset=+2]
+
+[id="risk-deployment-details-tab"]
+== Deployment details tab
```

Comment:
If the tab label indeed reads as "Deployment Details", I would capitalize Details in the header. Same for the "Risk Indicators tab" section.